libsolidity/analysis/ContractLevelChecker.cpp

```diff
@@ -51,6 +51,8 @@ bool hasEqualNameAndParameters(T const& _a, B const& _b)
 
 bool ContractLevelChecker::check(ContractDefinition const& _contract)
 {
+	_contract.annotation().unimplementedDeclarations = std::vector<Declaration const*>();
```

Comment:
Would it be possible to change the checking functions to return the unimplemented declarations instead?

Reply:
Ah ok, maybe just leave it...